pubspec.yaml

```diff
@@ -98,4 +98,4 @@ flutter:
   #         weight: 700
   #
   # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/custom-fonts/#from-packages
+  # see https://flutter.dev/custom-fonts/#from-packages
```

Comment:
nit: file shouldn't be changed here

Reply:
I think this happened when I was messing around to see if it was a dependency change which was caused the tests to fail. I'll fix this as well.